content/international-candidates.md

```diff
@@ -52,217 +58,144 @@ keywords:
 
-[Search for UK government-approved postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+#### An offer to train
 
-Not all training providers can accept international candidates. Contact your chosen teacher training providers directly, before you apply online, to get their advice about your eligibility and the application process.
+Explore different [ways to train](/ways-to-train) and consider whether you want do a training programme which leads to [qualified teacher status (QTS)](/ways-to-train). QTS allows you to apply to all teaching jobs in England.
 
-## How to apply for a visa to work or study in the UK
+Fees may be higher for international trainees but you may be able to [get financial support](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-non-uk-applicants#financial-support-for-non-uk-applicants-for-unsalaried-teacher-training-in-england).
 
-The UK introduced a [points-based immigration system](https://www.gov.uk/guidance/new-immigration-system-what-you-need-to-know) on 1 January 2021.
+Contact providers before you [apply](#apply-for-a-place) to get their advice about eligibility and the application process.
 
-### EU Settlement Scheme
+#### The right visa or status
 
-If you are a citizen of a country inside the [European Economic Area](https://www.gov.uk/eu-eea) (EEA) or Switzerland, and were living in the UK before 31 December 2020, you can apply to the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families).
+You need to [get the right visa or status to study in the UK](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-non-uk-applicants#visas-and-immigration).
 
-If your application is successful, you can continue to live, work and study in the UK after 30 June 2021. You will not need to use the points-based immigration system explained here.
+You usually need a confirmed offer on a course to apply for a visa.
 
-### Changes to the UK’s immigration system
+[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
 
-The points-based immigration system applies to all non-UK citizens arriving in the UK from 1 January 2021, including those from the EEA or Switzerland.
+#### To pass certain checks
 
-### Irish citizens
+Your training provider will check you’re suitable for teaching in terms of your health, whether you're safe to work with children and your level of maths and English.
 
-Irish citizens can continue to enter, work and study in the UK under the [Commmon Travel Area](https://www.gov.uk/government/publications/common-travel-area-guidance/common-travel-area-guidance) as they did previously.
+### Get support
 
-## Choosing the correct visa for your job or course
+You can get an adviser for one-to-one support.
```

Comment:
Is this all applicants or only those for key subjects?

Reply:
For international candidates who aren't returners? All applicants. If they were a returner, the key subjects triage would apply.